Source/Core/Common/HttpRequest.h

```diff
@@ -35,10 +35,12 @@ class HttpRequest final
   void UseIPv4();
   void FollowRedirects(long max = 1);
   std::string EscapeComponent(const std::string& string);
-  Response Get(const std::string& url, const Headers& headers = {});
-  Response Post(const std::string& url, const std::vector<u8>& payload,
-                const Headers& headers = {});
-  Response Post(const std::string& url, const std::string& payload, const Headers& headers = {});
+  Response Get(const std::string& url, const Headers& headers = {},
+               bool allow_bad_response_code = false);
```

Comment:
This bool parameter should likely be an enum class to avoid the ambiguous `true` specifiers at call sites.

Reply:
Fair enough.